File: term2/week1-rs/exercise9.rs
fn main() {
    for i in 25..51 {
        if is_prime(i) == true {
            println!("{}", i)
        }
    }
}

fn is_prime(n: u32) -> bool {
    if n <= 1 {
        return false;
    }
    for a in 2..n {
        if n % a == 0 {
            return false;
        }
    }
    return true
}

File: term2/week1-py/exercise9.py
def isprime(num):
    if num > 1:  
        for n in range(2,num):  
            if (num % n) == 0:  
                return False
        return True
    else:
        return False

for i in range(25, 51):
    if isprime(i):
        print(i)
